src/main/java/ca/uhn/fhir/jpa/starter/AppProperties.java

```diff
@@ -71,6 +71,7 @@ public class AppProperties {
   private List<String> supported_resource_types = new ArrayList<>();
   private List<Bundle.BundleType> allowed_bundle_types = null;
   private Boolean narrative_enabled = true;
+  private String history_count_mode = "CACHED_ONLY_WITHOUT_OFFSET";
```

Comment:
This should be bound to HistoryCountModeEnum like in https://github.com/hapifhir/hapi-fhir-jpaserver-starter/blob/e2d2ea7726975e76010fc3f98149fad72a48fe84/src/main/java/ca/uhn/fhir/jpa/starter/AppProperties.java#L67 making it more strongly typed